pkg/client/src/app/common/KeycloakProvider.tsx

```diff
@@ -1,9 +1,10 @@
 import { initInterceptors } from "@app/axios-config";
+import { ENV } from "@app/Constants";
 import i18n from "@app/i18n";
 import keycloak from "@app/keycloak";
 import { Flex, FlexItem, Spinner } from "@patternfly/react-core";
 import { ReactKeycloakProvider } from "@react-keycloak/web";
-import React from "react";
+import React, { useEffect } from "react";
```

Comment:
It's a detail, but since we're importing the whole, why bother and not just use React.useEffect() ?

Reply:
Very subjective - but I think destructuring makes the codebase cleaner. That way we don't have to litter our component with 'React dot' and instead can just refer to the method names.

Reply:
I think destructuring is actually a good thing and we should actually favor it to potentially allow better optimization for the bundle. What matter for the project is to be as consistent as possible, either way.